defold-spine/src/comp_spine_model.cpp

```diff
@@ -299,10 +299,10 @@ namespace dmSpine
 
     static void ClearCompletionCallback(SpineAnimationTrack* track)
```

Comment:
I use the same clean function to make sure we don't break current flow anywhere.